### ISSUE TITLE
Make network speed widget a fixed size

### ISSUE THREAD

**Issue by RedBearAK** [JaKooLit/Fedora-Hyprland#10]
On the Waybar style I've been using, there is a network info widget that keeps changing its width as the number of digits in the network speeds changes. This makes it very difficult to interact with the system tray icons. 

I'm sure there's a simple way to disable it entirely, but if there is a way to force that widget to adopt a fixed size, that would be nice. 

Very minor issue.

**Comment by RedBearAK** [comment]
Actually it looks like some of the other widgets like the CPU percentage on the left do the same kind of thing. But it's more of a problem with the network widget, since the tray icons are on its left side and keep getting moved around.

**Comment by JaKooLit** [comment]
its not an issue but thats because the the network keep changing... you can remove that module or choose another waybar layout 

super ctrl b layouts and super alt b for styles

that network widget is only on the default layout and not present

you can costumize the modules as much as you wanted

**Comment by RedBearAK** [comment]
@JaKooLit 

> you can costumize the modules as much as you wanted

True. There are an overwhelming number of choices. Would be nice to have a gallery of images that showcases all the waybar layouts and styles. I scanned through the README and didn't see anything like that.

**Comment by JaKooLit** [comment]
hmmmmmm I can but that would mean like more than more than hundreds of screenshots....

cause any styles, you can combine or apply to any layout........

**Comment by JaKooLit** [comment]
https://github.com/JaKooLit/Hyprland-Dots/wiki

maybe you want to take a look here as well...

**Comment by RedBearAK** [comment]
> more than more than hundreds of screenshots

No, just pick a default style and show all the different layouts, then show the styles separately with a single layout. You wouldn't need to show all the possible combinations. People will be able to figure out how to pair the layout they want with the style they like. But just going by name and having to try them all to see them and then remember which ones you liked, it's a bit hard. Especially if you have ADHD and a poor working memory.
